### ISSUE TITLE
Error on upgrade to 2.2.0

### ISSUE THREAD

**Issue by lloydwatkin** [seuros/capistrano-sidekiq#298]
```ruby
00:03 sidekiq:install
      Uploading /tmp/sidekiq.service 100.0%
      Uploading /tmp/sidekiq.service 100.0%
      Uploading /tmp/sidekiq.service 100.0%
      Uploading /tmp/sidekiq.service 100.0%
      01 sudo mv /tmp/sidekiq.service /etc/systemd/system/sidekiq.service
    ✔ 01 ubuntu@34.245.223.184 0.143s
      02 sudo systemctl daemon-reload
    ✔ 01 ubuntu@54.194.254.253 0.153s
    ✔ 01 ubuntu@34.253.22.235 0.155s
    ✔ 01 ubuntu@54.246.20.184 0.163s
    ✔ 02 ubuntu@34.253.22.235 0.297s
    ✔ 02 ubuntu@34.245.223.184 0.319s
      03 sudo systemctl enable sidekiq@{1..2}
    ✔ 02 ubuntu@54.194.254.253 0.322s
    ✔ 02 ubuntu@54.246.20.184 0.296s
      03 Failed to enable unit: Unit file sidekiq@1.service does not exist.
      03 Failed to enable unit: Unit file sidekiq@1.service does not exist.
    ✘ 03 ubuntu@34.245.223.184 0.154s
```

Also, what are the plans/timeline for fully automatic multiple process support using this library?

Thanks for the efforts :)

**Comment by lloydwatkin** [comment]
Ok, if I set `:sidekiq_processes` to 1, then I don't get an issue, its only if `sidekiq_processes` > 1 (after having upgraded to **2.2.0**).

I would make sense to have a consistent naming convention regardless of the number of processes?

**Comment by lloydwatkin** [comment]
(sorry for all the comments, just dumping my investigation....)

From what I can see the `start`, `stop`, `restart` processes are all covered, just not the `install` process. I believe I think this means that we need to update https://github.com/seuros/capistrano-sidekiq/blob/be3e88c5b3d7f5dcb3e1fa6dba6dceb3c8432ed0/lib/capistrano/tasks/systemd.rake#L153-L161 to take into account process numbers

**Comment by seuros** [comment]
Should be fixed in 2.3.0 . Please confirm .
